1-js/05-data-types/02-number/article.md

```diff
@@ -159,14 +159,14 @@ There are two ways to do so.
     alert( num.toFixed(1) ); // "12.3"
     ```
 
-    The rounding goes to the nearest value, similar to `Math.round`:
+    This round up or down to the nearest value, similar to `Math.round`:
```

Comment:
Maybe "This rounds"? The original wording is wrong?

Reply:
should be "This rounds..."

Reply:
And the original wording is wrong, @jmbothe ?

Reply:
No, the original wording is not wrong, but its awkward to use a verbal noun when a verb would suffice. But from a purely grammatical standpoint, the original wording is acceptable.